fvm/transactionInvocator.go

```diff
@@ -199,7 +228,7 @@ func (i *TransactionInvocator) Process(
 		Int("retried", proc.Retried).
 		Msg("transaction executed successfully")
 
-	return nil
+	return txError
```

Comment:
this should be nil, if we have a case that we are logging that was successful but the txError is not nil then we have a problem.